/pupil_glasses_node:
  ros__parameters:
    publish_freq: 50
    draw_circle: false
    camera_depth: 1.0
    video_resolution:
      - 1920
      - 1080
    ip: "192.168.0.2"
    port: "8080"
    print_performance: False

/aruco_detect:
  ros__parameters:
    camera_alpha: 0.0
    front_camera_image: pupil_glasses/front_camera/image_color
    front_camera_params: pupil_glasses/front_camera/camera_info
    publish_drawn_marker: True
    marker_length: 0.035
    aruco_dict: 6 # DICT_4X4_100
# Properties at https://docs.opencv.org/3.4/d9/d6a/group__aruco.html

/diegetic_button:
  ros__parameters:
    button_map_path: "src/diegetic_button_pkg/button_maps/ButtonMap - A3Piano.csv"
    button_center_finding_mode: "inverse_quintic"

/process_inputs_node:
  ros__parameters:
    input_trigger_mode: "sloppy"
    cycle_duration_seconds: 0.30 # seconds to complete transition
    active_threshold_percent: 0.40 # percentage. Close to 0 start quickly
    inactive_threshold_percent: 0.60 # percentage. Close to 1 stop quickly
    publish_image: True
    draw_on_fiducials: True
    serial_com: False
    send_serial: False

/process_inputs_node:
  ros__parameters:
    camera_matrix:
      [
        890.58737517,
        0.0,
        843.03517309,
        0.0,
        904.24555983,
        646.32207425,
        0.0,
        0.0,
        1.0,
      ]
    dist_coeff:
      [
        -2.42124213e-01,
        -1.08320751e+00,
        -3.75766513e-03,
        8.55747541e-04,
        4.68416642e+00,
      ]
